calibration/SPKIRB/CGINS-SPKIRB-00273__20160219.csv

```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+273,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR_OCR-507_SN_273_Calibration_Files_2016-02-19.zip > DI7273C.cal
+273,CC_offset,"[2147434564.6, 2147320957.5, 2146986105.6, 2147299570.4, 2147941911.2, 2147853518.7, 2146960907.9]",
+273,CC_scale,"[0.000000202683798991, 0.000000196954380415, 0.000000209925813595, 0.000000202472495602, 0.000000214775095274, 0.00000021177099307, 0.000000203138408179]",
```

Comment:
Checked. All good. Again, are we going with scientific notation or no? There have been both uploaded.

Reply:
The excel macro that Rutgers (?) created formats this way.  They said it doesn't matter - other than that it isn't as easy to read or as clean.

Reply:
Looks good.